Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeFormUIModel.swift

```diff
@@ -190,7 +190,12 @@ class CreateSafeFormUIModel {
 
     func setChain(_ scgChain: SCGModels.Chain) {
         let newChain = Chain.createOrUpdate(scgChain)
+        if chain.id != newChain.id {
```

Comment:
how can this happen?

Reply:
If you select same chain again in the UI

Reply:
why would it have different id then?

Reply:
Here I'm trying to prevent the change if the user selected the same chain again and again 

>